### PR TITLE
add contribution guidelines

### PR DESCRIPTION
**Pull request by zaibon**
part of #240

I got inspiration from the very nice contribution guideline line of [Istio](https://github.com/istio/community/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by muhamadazmy — APPROVED** [review]

**Review by delandtj — APPROVED** [review]
lgtm

File: VALUES.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Team Values
+
+We want to make sure every member has a shared understanding of the goals and
+values we hold as a team:
+
+- Optimize for the **overall project**, not your own area or feature
+  - A shortcut for one individual can mean a lot of extra work or disruption for
+    the rest of the team.
+
+- Our repos should always be in release shape: **Always Green**
```
Comment (maxux) [inline VALUES.md]: I would precise `master branch`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+We promote and encourage a set of [shared values](VALUES.md) to improve our
+productivity and inter-personal interactions.
+
+## Contributing a feature
+
+In order to contribute a feature you'll need to go through the following steps:
+
+- Create a Github issue to explain the feature you want to create. A member of the core team will engage with
+you and together you will decide if the feature could be accepted and what is the best approach to take to implement it.
```
Comment (maxux) [inline CONTRIBUTING.md:23]: Should't the label `type_feature` at least set ?

Comment (zaibon) [reply CONTRIBUTING.md:23]: normal user cannot set labels on issue. This is only possible for owner/contributors

File: VALUES.md (hunk 1 above)
Comment (zaibon) [reply VALUES.md]: indeed

**Review by maxux — APPROVED** [review]

**Pull request merged**